charts/exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: {{ $.Release.Namespace | quote }}
+  name: {{ include "releasename" $ | quote }}
+  labels: {{- include "labels" $ | nindent 4 }}
+  annotations: {{- toYaml $.Values.annotations | nindent 4 }}
+spec:
+  selector:
+    matchLabels: {{- include "matchLabels" $ | nindent 6 }}
+  replicas: {{ int $.Values.replicaCount }}
+  strategy:
+    type: RollingUpdate
+  revisionHistoryLimit: 3
+
```

Comment:
```suggestion
```
nitpick: empty line that I don't think aids readability

Reply:
Fixed in 1dc956030625c8509df32179ddaca42fa90c4f2c